server/index.js

```diff
@@ -183,6 +161,28 @@ export default class Server {
         await this.serveStatic(req, res, p)
       },
 
+      '/_next/:buildId/page/:path*.js.map': async (req, res, params) => {
+        const paths = params.path || ['']
+        const page = `/${paths.join('/')}`
+
+        if (this.dev) {
+          try {
+            await this.hotReloader.ensurePage(page)
+          } catch (error) {}
+
+          const compilationErr = await this.getCompilationError()
```

Comment:
We don't need to a HTML error for this. Having just the status code fine.
I think we don't need to use `try catch` at all.

Reply:
Hey make sure remove this unnecessary code.